### DISCUSSION TITLE
Having issues getting server to launch

### DISCUSSION THREAD

**Discussion by Wolfipoo** [gekigek99/minecraft-vanilla-server-hibernation#127] [Help required]
I believe I have everything configured correctly, but every time I join the server checking if it'll start it up, it gives me "Minecraft server is starting" and at the same time "Minecraft server is offline" with no other errors. I was having issues with getting Java to work correctly, but I believe that issue is fixed and this is something else. Using 2.4.4 currently.

```
{

  "Server": {
    "Folder": "C:\\Users\\darkm\\Desktop\\MC Server",
    "FileName": "fabric-server-launch.jar",
    "Protocol": 757,
    "Version": "1.18.1"
  },
  "Commands": {
    "StartServer": "java -jar fabric-server-launch.jar nogui",
    "StartServerParam": "-Xmx8192M -Xms8192M",
    "StopServer": "stop",
    "StopServerAllowKill": 10
  },
  "Msh": {
    "Debug": 1,
    "InfoHibernation": "                   §fserver status:\n                   §b§lHIBERNATING",
    "InfoStarting": "                   §fserver status:\n                    §6§lWARMING UP",
    "NotifyUpdate": true,
    "ListenPort": 25555,
    "TimeBeforeStoppingEmptyServer": 60
  }
}
```

**Comment by gekigek99** [comment]
from what I see the `start server` should look like `"StartServer": "java <Commands.StartServerParam> -jar <Server.FileName> nogui"` _(don't remove the `<>` stuff)_

so if you go in this folder `C:\\Users\\darkm\\Desktop\\MC Server` and execute `java -Xmx8192M -Xms8192M -jar fabric-server-launch.jar nogui` does the server start?

try setting the `debug` to `3`, launch msh and type `msh start` command (the server should do a manual start): report here the log that comes out of that.
